vm/src/builtins/str.rs

```diff
@@ -1017,21 +1017,31 @@ impl PyStr {
         !self.data.is_empty() && self.char_all(char::is_alphabetic)
     }
 
+    /// Return a copy of the string with all occurrences of substring old replaced by new.
+    /// If count is given, only the first count occurrences are replaced.
+    /// If count is not specified or -1, then all occurrences are replaced.
```

Comment:
```suggestion
```
We have a special document handling for `pymethod`. It automatically copies `__doc__` from Python doc database https://github.com/RustPython/__doc__.
But if we add this comment, this one will have priority

Reply:
oh, I just looked at https://github.com/RustPython/RustPython/blob/e75aebb967266c02c2b10f9294afe2b8df878efc/vm/src/builtins/str.rs#L1038-L1052
And thought I might as well, so the docs from `str.ispritable` needs to be removed?